Sources/App/Services/Helper/DefaultQuestionnaireResponseLoader.swift

```diff
@@ -0,0 +1,40 @@
+//
+// This source file is part of the ENGAGE-HF-AI-Voice open source project
+//
+// SPDX-FileCopyrightText: 2022 Stanford University and the project authors (see CONTRIBUTORS.md)
+//
+// SPDX-License-Identifier: MIT
+//
+
+import Foundation
+import ModelsR4
+import Vapor
+
+
+/// Default implementation of QuestionnaireResponseLoader
+struct DefaultQuestionnaireResponseLoader: QuestionnaireResponseLoader {
```

Comment:
We will eventually have to support different responses per phone number. Ideally we even have a mapping of phone number to user ID. We won't have this at the beginning but was we integrate with the production system we will probably have a nice mapping to user ID and should aim to use that in future versions of the system.